flang/test/Semantics/OpenMP/omp-atomic-assignment-stmt-read.f90

```diff
@@ -0,0 +1,43 @@
+! RUN: %python %S/../test_errors.py %s %flang -fopenmp
```

Comment:
You are not testing for any errors here, I think you are just testing that flang will compiler this file correctly. Using the test_errors.py script is incorrect because it allows flang to fail.

Reply:
Removed usage of test_errors.py from the test case.